cnv/cnv_release_notes/cnv-release-notes.adoc

```diff
@@ -17,6 +17,13 @@ include::modules/technology-preview.adoc[leveloffset=+2]
 
 // CNV-2928/CNV-4166 OVN-Kubernetes network provider
 
+// CNV-3535 Dedicated resources
+* A new option to view and configure dedicated resources for virtual machines is
+available in the web console *Details* view. When you enable dedicated
+resources, you ensure that your virtual machine's workload runs on CPUs that
+are not used by other processes, improving your virtual machine's performance
```

Comment:
Sorry, should have been "...other processes. This can improve virtual machine performance..."

Reply:
No probs. This now reads:

=== Web console
// CNV-4076 UI view/config dedicated resources
* A new option to view and configure dedicated resources for virtual machines is
available in the web console *Virtual Machine Details* view. When you enable dedicated
resources, you ensure that your virtual machine's workload runs on CPUs that
are not used by other processes. This can improve your virtual machine's performance
and the accuracy of latency predictions.